<p>Please send URL of your R software, and I am more than happy to include it here.</p>
<p><strong>SOFTWARE (<a href="http://cran.r-project.org/web/views/Genetics.html">CRAN task view for genetics</a> and <a href="http://www.bioconductor.org/">www.bioconductor.org</a>)</strong></p>
<ul>
<li><a href="http://adegenet.r-forge.r-project.org/">adegenet</a> by T Jombart</li>
<li><a href="http://www.cancerbiostats.onc.jhmi.edu/BayesMendel/">BayesMendel</a></li>
<li><a href="http://cnv-tools.sourceforge.net/">cnv-tools</a> </li>
<li><a href="http://www-gene.cimr.cam.ac.uk/clayton/software/">dgc.genetics</a> by D Clayton</li>
<li><a href="http://statgen.ncsu.edu/zaykin/Dprime/">Dprime</a> by D Zaykin</li>
<li><a href="http://linkage.rockefeller.edu/yyang/resources.html">EHP</a> (Estimating haplotypes for pooled DNA sample)</li>
<li><a href="http://mga.bionet.nsc.ru/nlru/GenABEL/">GenABEL</a> by Yurii Aulchenko</li>
<li><a href="http://homepages.lshtm.ac.uk/~encdcver/">graphminer</a> by Claudio Verzili </li>
<li><a href="http://www.nr.no/pages/samba/area_emr_smbi_gseasnp">GSEA-SNP</a></li>
<li><a href="http://www4.stat.ncsu.edu/~jytzeng/Softwares/Hap-Clustering/R/">Hap-Clustering</a> by JY Tzeng</li>
<li><a href="http://www.uib.no/smis/gjessing/genetics/software/haplin/">HAPLIN</a> by HK Gjessing</li>
<li><a href="http://www.well.ox.ac.uk/~rmott/happy.html">HAPPY</a> by R Mott</li>
<li><a href="http://www.stat.auckland.ac.nz/~browning/HapVLMC/index.htm">HapVLMC</a> by SR Browning</li>
<li><a href="http://fisher.utstat.toronto.edu/~shuying/research/research.html">HMM.hap</a> by Shuying Sun</li>
<li><a href="http://faculty.washington.edu/jonno/software.html">HWEBAYES</a> by Jon Wakefield </li>
<li><a href="http://www.fumihiko.takeuchi.name/publications.html">LDGROUP</a> by Takeuchi et al.</li>
<li><a href="http://www.uhnresearch.ca/labs/tritchler/">LINK/TDT</a> by D Tritchler</li>
<li><a href="http://pbil.univ-lyon1.fr/~rezvoy/mareymap/index.php">MareyMap</a> by Clement Rezvoy</li>
<li><a href="http://mayoresearch.mayo.edu/mayo/research/biostat/splusfunctions.cfm">multic</a> by M de Andrade, E Lunde, E Atkinson</li>
<li><a href="http://csg.sph.umich.edu/boehnke/p_act.php">p_act</a> by Michael Boehnke</li>
<li><a href="http://wpicr.wpic.pitt.edu/wpiccompgen/">Packages</a> by B Devlin &amp; K Roeder</li>
<li><a href="http://cancercenter.mayo.edu/mayo/research/biostat/schaid.cfm">Packages</a> by D Schaid</li>
<li><a href="http://www.biostat.wisc.edu/~kbroman/software/">Packages</a> by K Broman</li>
<li><a href="http://www.pedvizapi.org/">PedVizAPI</a></li>
<li><a href="http://www.qtlbim.org/">QTLBIM</a></li>
<li><a href="http://www.fhcrc.org/science/labs/tang/">SABER</a> by H Tang Score statistics by K Wang</li>
<li><a href="http://www.sph.umich.edu/csg/abecasis/Exact/r_instruct.html">SNP-HWE</a> by G Abecasis</li>
<li><a href="http://www.cs.uiowa.edu/~jian/">Software</a> by J Huang</li>
<li><a href="http://www.stat.purdue.edu/~jinj/Research/software/">Software</a> by Jiashun Jin </li>
<li><a href="http://bear.fhcrc.org/~clk/soft.html">Software</a> by Charles Kooperberg </li>
<li><a href="http://www.cs.toronto.edu/~radford/software-online.html">Software</a> by Radford Neal </li>
<li><a href="http://faculty.washington.edu/jonno/cv.html">Software</a> by Jon Wakefield </li>
<li><a href="http://www.hsph.harvard.edu/faculty/kraft/soft.htm">Software</a>by P Kraft</li>
<li><a href="http://www.msbi.nl/dnn/">Software</a> from Leiden University Medical Center</li>
<li><a href="http://www.statisticalsource.com/software/statsoftware.htm">Software</a> from Statistical Source </li>
<li><a href="http://www.mpipsykl.mpg.de/pages/english/research/mueller_downloads.htm">Software</a> by Muller-Myhsok</li>
<li><a href="http://www.mskcc.org/mskcc/html/19904.cfm">Software for two-stage design</a> by J Satagopan</li>
<li><a href="http://www.vipbg.vcu.edu/~edwin/">Software</a> by Edwin JCG Van den Oord </li>
<li><a href="http://www.biostat.harvard.edu/~betensky/ahg.programs">TDTexact</a> by R Betensky</li>
</ul>
<p><strong>STANDARD LINKS</strong></p>
<ul>
<li><a href="https://r-forge.r-project.org/">R-Forge</a></li>
<li><a href="http://cran.r-project.org/">The Comprehensive R Archive Network</a> (CRAN)</li>
<li><a href="http://wiki.r-project.org/rwiki/doku.php">The R wiki website</a></li>
<li><a href="http://www.insightful.com/">The S-PLUS home page</a></li>
<li><a href="http://rgenetics.org/">http://rgenetics.org</a> </li>
<li><a href="http://www.rforge.net/">rforge.net</a></li>
</ul>
<p><strong>PUBLICATIONS</strong></p>
<p><a href="http://www.r-project.org/">R</a></p>
<ul>
<li>Joo J, Kwak, M, Zheng G. Improving power for testing genetic association in case-control studies by reducing the alternative space. Biometrics 2010; 66:266-276</li>
<li>Wakefield J. Bayesian Methods for Examining Hardy-Weinberg Equilibrium. Biometrics 2010; 66:257-65</li>
<li>Wakefield J. Bayes Factors for Genome-Wide Association Studies: Comparison with P-values.Genet Epidemiol 2009;33:79-86</li>
<li>Holden et al. GSEA-SNP: applying gene set enrichment analysis to SNP data from genome-wide association studies. Bioinformatics 2009;24:2784-5</li>
<li>Kitamura, et al. Programs for calculating the statistical powers of detecting susceptibility genes in case-control studies based on multistage designs. Bioinformatics 2009;25:272-3</li>
<li>Foulkes AS. <em>Applied Statistical Genetics with R for Population-based Association Studies</em>. Springer 2009</li>
<li>Guillot G, Santos F, Estoup A. Analysing georeferenced population genetics data with Greenland: a new algorithm to deal with null allles and a friendly graphical user interface. Bioinformatics 2008;24:1406-7</li>
<li>Jombart T.adegenet: a R package for the multivariate analysis of genetic markers. Bioinformatics 2008;24:1403-5</li>
<li>Siegmund D, B Yakir. <em>The Statistics of Gene Mapping</em>. Springer 2007</li>
<li>Wu R, C-X Ma, G Casella.<em>Statistical Genetics of Quantitative Traits-Linkage, Maps and QTL</em>. Springer 2007</li>
<li>Rezvoy C, Charif D, Gueguen L, Marais GAB. MareyMap: an R-based tool with graphical interface for estimating recombination rates. Bioinformatics 2007;23:2186-9</li>
<li>Yandell BS, Mehta T, Banerjee S, Shriner D, Venkataraman R, Moon JY, Neely WW, Wu H, Smith R, Yi N. R/qtlbim: QTL with Bayesian Interval Mapping in experimental crosses. Bioinformatics 2007; 23(5):641-643</li>
<li>Kooperberg C, M LeBlanc. Increasing the power of identifying gene x gene interactions in genome-wide association studies. Genet Epidemiol 2008;32:255-63</li>
<li>Zhao JH. gap: genetic analysis package. J Statist Soft 2007; 23(8):1-18</li>
<li>Sun S, CMT Greeenwood, RM Neal. Haplotype inference using a Bayesian hidden Markov model. Genet Epidemiol 2007;31:937-48</li>
<li>Clayton D., Leung,H.T. An R package for analysis of whole-genome association studies. Hum Hered 2007; 64:45-51</li>
<li>Aulchenko YS, Ripke S, Isaacs A, van Duijn CM (2007). GenABEL: An R Library for genome-wide association analysis. Bioinformatics 2007; 23(10):1294-6</li>
<li>Gonzalez,J.R., Armengol,L., Sole,X., Guino,E., Mercader,J.M., Estivill,X., Moreno,V. SNPassoc: an R package to perform whole genome association studies. Bioinformatics 2007; <strong>23</strong>: 644-645 areast-language: ZH-CN; mso-bidi-language: AR-SA"}</li>
<li>Shin J-H, McNeney B, Graham J. Case-control inference of interaction between genetic and nongenetic risk factors under assumptions on their distribution. Statist Appl Genet Mol Biol 2007; 6(1):13</li>
<li>Estimating the null and the proportion of non-null effects in large-scale multiple comparisons. JASA 2006; <strong>102</strong>:495-506</li>
<li>Hoffmann T, Lange C. P2BAT: a massive parallel implementation of PBAT for genome-wide association studies in R. Bioinformatics 2006; 22(24):3103-5</li>
<li>Shin J-H, Blay S, McNeney B, Graham J. LDheatmap: An R function for graphical display of pairwise linkage disequilibria between single nucleotide polymorphisms. J Statist Soft 2006; 16 (3):1-9</li>
<li>Burkett K, Graham J, McNeney B. happassoc: software for likelihood inference of trait associations with SNP haplotype and other attributes. J Statist Soft 2006; 16(2):1-19</li>
<li>Zhao JH, Tan Q. Genetic dissection of complex traits <em>in silico</em>: approaches, problems and solutions. Current  Bioinformatics 2006; 1(3):359-69</li>
<li>Tan H, M Coram, P Wang, X Zhu, N Risch. Reconstructing genetic ancestry blocks in admixed individuals. Am J Hum Genet 2006; <strong>79</strong>:1-12</li>
<li>Zhao JH, Tan Q. Integrated analysis of genetic data with R.Hum Genomics. 2006 Jan;2(4):258-65</li>
<li>Tufto J, Cavallini P. Should wildlife biologists use free software? Wildlife Biology 2005; 11(1):67-76</li>
<li>Gentleman R, V Carey, W Huber, R Irizarry, S Dudoit. <em>Bioinformatics and Computational Biology Solutions Using R and Bioconductor</em>. Springer 2005</li>
</ul>
<p><strong><a href="http://www.sas.com/">SAS</a></strong></p>
<ul>
<li>Zhao JH, Luan JA, Tan Q, Loos R, Wareham N. Analysis of Large Genomic Data <em>in Silico</em>: The EPIC-Norfolk Study of Obesity. Huang, D.-S., Heutte, L., and Loog, M. Communication in Computer and Information System [2], 781-790. 2007.  Springer-Verlag Berlin Heidelberg</li>
<li>Saxton AM. (Ed) <em>Genetic Analysis of Complex Traits Using SAS</em>. 2004, SAS Publications  ISBN:  1-59047-507-0 (#154)</li>
</ul>
<p><em>Created</em> 9/3/2006, <strong><em>Last changed</em></strong> 19/12/2018</p>
